docs/devguide/newdashboards.asciidoc

```diff
@@ -72,7 +72,7 @@ Flags:
   -h, --help               help for setup
       --machine-learning   Setup machine learning job configurations only
       --modules string     List of enabled modules (comma separated)
-      --template           Setup index template only
+      --index-management           Setup index template only
```

Comment:
Please also update the explanation to something like `Setup index management only (templates and ILM policies)`.

Reply:
Looking at this more closely, if we're going to show the output of running  `metricbeat help setup`, the example should show the exact output of running this command.